### PR TITLE
User interface

### PR DESCRIPTION
**Pull request by kalifraser**

### CODE REVIEW

**Comment by kalifraser** [comment]
Code to get the GUI working and integrated into neural net. Edits to evaluateNN are to integrate GUI and to work with the most recent version of Keras (np_utils was merged into utils in the current version)

File: NeuralNet/evaluateNN.py [hunk 1]
```diff
@@ -79,15 +79,15 @@ def parse_input(filename):
                                                                        axis=1, ignore_index=True),
                                                              frac=1), drop=True)
 train_labels = shuffled_data[20]
-train_data = shuffled_data.drop(columns=20)
+train_data = shuffled_data.drop(columns=[20])
```
Comment (mynktwri) [inline NeuralNet/evaluateNN.py]: Is putting this in brackets okay?

File: NeuralNet/evaluateNN.py [hunk 2]
```diff
@@ -133,3 +133,29 @@ def parse_input(filename):
 results = model.evaluate(test_data, test_labels)
 
 print(results)
+
+def makePrediction(sentence):
+    (indices, wordlist, poslist) = db_pull.in_pipe([sentence])
```
Comment (mynktwri) [inline NeuralNet/evaluateNN.py]: id suggest changing these variable names cus they conflict with same names in a bigger scope

File: NeuralNet/evaluateNN.py (hunk 1 above)
Comment (kalifraser) [reply NeuralNet/evaluateNN.py]: It runs fine, so yes? I don't think it makes a big difference

File: NeuralNet/evaluateNN.py (hunk 2 above)
Comment (kalifraser) [reply NeuralNet/evaluateNN.py]: I'm not sure where the conflict is. Can you change them?

**Pull request merged**